e2e/specs/register-new-patient.spec.ts

```diff
@@ -25,23 +24,63 @@ test('Register a new patient', async ({ page }) => {
     stateProvince: 'Pernambuco',
     cityVillage: 'Recife',
     phone: '5555551234',
+    email: 'johnnyronny@example.com',
   };
 
   await test.step('When I visit the registration page', async () => {
     await patientRegistrationPage.goto();
     await patientRegistrationPage.waitUntilTheFormIsLoaded();
   });
 
-  await test.step('And then I fill the registration form with the data in `formValues` and then click the `Submit` button', async () => {
-    // Check for explicit visibility of address template fields
+  await test.step('And I fill in the first name', async () => {
+    await patientRegistrationPage.givenNameInput().fill(formValues.givenName);
+  });
+
+  await test.step('And I fill in the middle name', async () => {
+    await patientRegistrationPage.middleNameInput().fill(formValues.middleName);
+  });
+
+  await test.step('And I fill in the family name', async () => {
+    await patientRegistrationPage.familyNameInput().fill(formValues.familyName);
+  });
+
+  await test.step('And I fill in the gender', async () => {
```

Comment:
The value you enter should also be included in the step. The goal is for someone to understand what the step does just by looking at it in the report, without needing to check the source. Take a look at the steps in the Register an unknown patient test.